webserver/views/api/v1/core.py

```diff
@@ -6,21 +6,21 @@
 from webserver.decorators import crossdomain
 import webserver.exceptions
 import json
-import uuid
 
-api_bp = Blueprint('api', __name__)
+bp_core = Blueprint('api_v1_core', __name__)
 
 
-@api_bp.route("/<uuid:mbid>/count", methods=["GET"])
+@bp_core.route("/<uuid:mbid>/count", methods=["GET"])
 @crossdomain()
 def count(mbid):
+    """Endpoint for getting number of submissions for a given recording."""
     return jsonify({
         'mbid': mbid,
-        'count': count_lowlevel(mbid),
+        'count': count_lowlevel(str(mbid)),
     })
 
 
-@api_bp.route("/<string:mbid>/low-level", methods=["GET"])
+@bp_core.route("/<uuid:mbid>/low-level", methods=["GET"])
```

Comment:
I'd say move this page to sphinx completely.

It's supposed to be UUID (MBID). Now it's more self-documenting, looks more correct is these docs I added. I'm just doing the conversion below, as you can see.